packages/grafana-ui/src/utils/closePopover.ts

```diff
@@ -0,0 +1,13 @@
+export const closePopover = (event: React.KeyboardEvent<HTMLDivElement>, hidePopper: () => void) => {
```

Comment:
I think you should just be able to do without the type parameter - just event: React.KeyboardEvent` should do :)

Otherwise, maybe the `HTMLElement` type instead for this more generic component. `HTMLElement` is the base type that all HTML elements extend from, so it makes this method slightly more flexible.